static/bidder-info/algorix.yaml

```diff
@@ -0,0 +1,8 @@
+maintainer:
+  email: "xunyunbo@algorix.co"
+capabilities:
```

Comment:
Just to confirm, is AlgoriX not registered with the IAB Europe for GDPR?

Reply:
AlgoriX not registered with the IAB Europe for GDPR.

Reply:
Fantastic. Kindly please update your adapter in the future when you have a GDPR vendor id.